server/views/partials/profileBanner/profileBannerTop.njk

```diff
@@ -43,21 +43,13 @@
     {% endif %}
     <div class="govuk-grid-row">
       <div class="govuk-grid-column-full hmpps-profile-banner-top">
-        {% if category == 'A' %}
+        <a href="/prisoner/{{ prisonerNumber }}/image" class="hmpps-clickable-image" data-qa="prisoner-photo-link">
           <img
             class="hmpps-profile-photo"
-            src="/api/prisoner/{{ prisonerNumber or photoType }}/image"
```

Comment:
Given prisonerNumber is always defined I actually think this never did anything 🤔